tests/src/tracing/eventpipe/eventsvalidation/ExceptionThrown_V1.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using Microsoft.Diagnostics.Tools.RuntimeClient;
+using Microsoft.Diagnostics.Tracing;
+using Tracing.Tests.Common;
+
+namespace Tracing.Tests.ExceptionThrown_V1
+{
+    public sealed class MyEvents
```

Comment:
You can probably do away with this class entirely and change the `_eventGeneratingAction` to just be 
```csharp
        private static Action _eventGeneratingAction = () => 
        {
            for (int i = 0; i < 1000; i++)
            {
                try
                {
                    throw new ArgumentNullException("Throw ArgumentNullException");
                } 
                catch (Exception e)
                {
                    //Do nothing
                }
            }
        };
```